src/app.config.js

```diff
@@ -30,7 +30,15 @@ const Config = {
     return validExtensions.indexOf('.' + fileExt) > -1;
   },
   emptyValue: '--',
-  maxTopAlarms: 5
+  maxTopAlarms: 5,
+
+  OPERATOR_OPTIONS: [
```

Comment:
Check with Molly.  Both of you have something similar to this.  If they are the same, then we should utilize the same code... and certainly not completely different approaches.

Reply:
fixed